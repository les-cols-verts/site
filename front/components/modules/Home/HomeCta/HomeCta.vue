<template>
  <section class="links">
    <app-link
      text="Qui sommes-nous ?"
      href="https://www.lescolsverts.com/collectifs/"
    ></app-link>
    <app-link
      text="Nos dispositifs d'agriculture urbaine"
      href="https://www.lescolsverts.com/services/"
    ></app-link>
    <app-link
      text="Découvrir nos actions nationales"
      href="https://www.lescolsverts.com/collectifs/"
    ></app-link>
    <app-link
      text="Nos actualités"
      href="https://www.lescolsverts.com/blog/"
    ></app-link>
    <app-link
      text="Nous contacter"
      href="https://www.lescolsverts.com/contact/"
    ></app-link>
  </section>
</template>
<script>
import AppLink from '@/components/common/Link/AppLink';

export default {
  name: 'HomeCta',
  components: { AppLink },
};
</script>
<style scoped>
.links {
  display: flex;
  flex-direction: column;
  max-width: 80vw;
  margin-bottom: 1em;
}

@media only screen and (min-width: 1024px) {
  .links {
    max-width: 50vw;
    text-align: center;
    margin-bottom: 0;
    margin-right: 1em;
  }
}
</style>
