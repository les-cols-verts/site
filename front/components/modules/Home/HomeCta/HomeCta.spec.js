import HomeCta from '@/components/modules/Home/HomeCta/HomeCta';

describe('HomeCta', () => {
  it('should match snapshot', () => {
    // When
    const wrapper = shallowMount(HomeCta);

    // Then
    expect(wrapper).toMatchSnapshot();
  });
});
