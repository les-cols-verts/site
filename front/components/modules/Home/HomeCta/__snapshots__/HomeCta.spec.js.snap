// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`HomeCta should match snapshot 1`] = `
<section class="links">
  <app-link-stub color="" backgroundcolor="" text="Qui sommes-nous ?" href="https://www.lescolsverts.com/collectifs/"></app-link-stub>
  <app-link-stub color="" backgroundcolor="" text="Nos dispositifs d'agriculture urbaine" href="https://www.lescolsverts.com/services/"></app-link-stub>
  <app-link-stub color="" backgroundcolor="" text="Découvrir nos actions nationales" href="https://www.lescolsverts.com/collectifs/"></app-link-stub>
  <app-link-stub color="" backgroundcolor="" text="Nos actualités" href="https://www.lescolsverts.com/blog/"></app-link-stub>
  <app-link-stub color="" backgroundcolor="" text="Nous contacter" href="https://www.lescolsverts.com/contact/"></app-link-stub>
</section>
`;
